src/index.ts

```diff
@@ -93,6 +93,8 @@ function createPreprocessor(
 			ignored: /(^|[/\\])(\.|node_modules[/\\])/,
 		});
 
+		if (config?.exclude) watcher.unwatch(config.exclude);
```

Comment:
Config is guaranteed to be an object.
```suggestion
		if (config.exclude) watcher.unwatch(config.exclude);
```